app/boards/shields/a_dux/Kconfig.defconfig

```diff
@@ -1,19 +1,41 @@
 # Copyright (c) 2021 The ZMK Contributors
 # SPDX-License-Identifier: MIT
 
+if SHIELD_A_DUX_LEFT || SHIELD_A_DUX_RIGHT
+
+if ZMK_BLE || ZMK_USB
```

Comment:
Why isn't this just defaulted on? Why behind this check?

Reply:
Yes, does not makes much sense. No idea why i added this.
Will remove this check.